src/test_lib_json/main.cpp

```diff
@@ -3309,15 +3309,15 @@ JSONTEST_FIXTURE_LOCAL(CharReaderAllowDropNullTest, issue178) {
       {__LINE__, R"({"a":,"b":true})", 2, objGetAnd("a", checkEq(nullValue))},
       {__LINE__, R"({"a":})", 1, objGetAnd("a", checkEq(nullValue))},
       {__LINE__, "[]", 0, checkEq(emptyArray)},
-      {__LINE__, "[null]", 1},
```

Comment:
Instead of changing several test lines, we can silence the warning with one change in the TestSpec definition, Line 3305.

-    ValueCheck onRoot;
+    ValueCheck onRoot{};

Reply:
I had try it.
But it would report errors, like:
```
error: could not convert ‘{3320, "[,,null]", 3}’ from ‘<brace-enclosed initializer list>’ to ‘TestCharReaderAllowDropNullTestissue178::runTestCase()::TestSpec’
```